js/ui/interaction.js

```diff
@@ -36,6 +36,7 @@ function Interaction(el) {
     scrollwheel(zoom);
     el.addEventListener('dblclick', ondoubleclick, false);
     window.addEventListener('resize', resize, false);
+    window.addEventListener('keydown', keydown, false);
```

Comment:
@jfirebaugh to track focused state and fire `focus` and blur` events https://github.com/Leaflet/Leaflet/blob/98bf92530f56c0bee1f0764e0934a25db6a6692d/src/map/handler/Map.Keyboard.js#L77-L85